src/app/core/services/usage/usage.service.ts

```diff
@@ -36,17 +36,23 @@ export class UsageService {
     })
   }
 
-  sendQuestionnaireEvent(type, task: Task) {
+  sendQuestionnaireEvent(
+    type,
+    taskName: string,
+    taskTimestamp: number,
+    metadata?: string
```

Comment:
according to suggested changes in schema, this needs to be a Map<String, String>